adr/0006-supported-node-and-npm-versions.md

```diff
@@ -0,0 +1,41 @@
+# Supported Node Versions
+
+Date: 2017-12-15
+
+## Status
+
+Proposed
+
+## Context 
+
+eLife has projects built in Node.
+ 
+eLife has projects that use the Node Package Manager (npm) in at least part of their own build process, even if the main technology of the project is not Node.
+
+In order to provide language version consistency across projects we need to get a consensus on which Node versions we are going to support.
+
+In order to provide dependency management consistency, we need a consensus on which npm versions to support.
+
+Staying up to date with the major and minor versions of Node is important:
+
+- to be able to use new features (nice to have)
+- to keep working on a version that receives bug fixes and security updates (must have)            
+
+## Decision
+
+To use 8.x, the current Long Term Support (LTS) line.
```

Comment:
@elifesciences/yld will this be enough also to run and contribute to xpub releases? From an operations point of view, the intention is to run everything as a container image, but local development is influenced by these choices.

Reply:
Yes, we're targeting the latest LTS release (currently 8.9).